pkg/operation/botanist/component/istio/istiod.go

```diff
@@ -20,20 +20,19 @@ import (
 	"path/filepath"
 	"time"
 
-	"github.com/gardener/gardener/pkg/chartrenderer"
-	"github.com/gardener/gardener/pkg/features"
-	gardenletfeatures "github.com/gardener/gardener/pkg/gardenlet/features"
-	"github.com/gardener/gardener/pkg/operation/botanist/component"
-	"github.com/gardener/gardener/pkg/utils/managedresources"
-
 	networkingv1alpha3 "istio.io/client-go/pkg/apis/networking/v1alpha3"
 	corev1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	crclient "sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/gardener/gardener/pkg/chartrenderer"
```

Comment:
Shouldn't the order of the imports be different or did I miss the memo ;-) ?

Reply:
This is the "newer" style that we use. First go imports, then external repo imports, then gardener imports :)

Reply:
This is the "newer" style that we use. First go imports, then external repo imports, then gardener imports :)